docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,62 @@
+# Release 403 (12 Nov 2022)
+
+## General
+
+* Add the amount of data read from external sources during table scans to
+  `EXPLAIN ANALYZE`. ({issue}`14907`)
+* Fix handling of `time(n) with time zone` when clients do not support variable
+  precision temporal types. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
+  function. ({issue}`14886`)
+* Fix failure for certain queries involving joins over partitioned tables. ({issue}`14317`)
+* Fix incorrect order of parameters in `DESCRIBE INPUT` when they appear in a
+  `WITH` clause. ({issue}`14738`)
+
+## Security
+
+* Allow access token passthrough when using OAuth 2.0 with refresh tokens
+  enabled. ({issue}`14949`)
+
+## Fault-tolerant execution
+
+* Override the values of the `exchange.compression-enabled`,
+  `query.low-memory-killer.delay`, `query.remote-task.max-error-duration`, and
+  `task.concurrency` configuration properties to the recommended values for
+  fault-tolerant execution when it is enabled. ({issue}`14957`)
+
+## BigQuery connector
+
+* Improve performance of `SHOW SCHEMAS` by adding a metadata cache. This can be
+  configured with the `bigquery.metadata.cache-ttl` catalog property, which has
+  a default value of `0ms` (disabled). ({issue}`14729`)
+* Fix failure when a [row access 
+  policy](https://cloud.google.com/bigquery/docs/row-level-security-intro)
+  returns an empty result. ({issue}`14760`)
+
+## ClickHouse connector
+
+* Add support for ClickHouse `DateTime(timezone)` type in read-only operations. ({issue}`13541`)
```

Comment:
It'd be good to let authors know when RNs are editorialized. Historically we've always mentioned the data type which a type is being mapped to in release notes - specially since we don't link to docs from here.

In this case the original from https://github.com/trinodb/trino/issues/14878#issuecomment-1303857828 seems better.

Reply:
When a mapping changes, that may impact existing workflows that rely on the old mapping. When a mapping can be used for writes, you need to know what type to use in Trino to write to the connector. Those are when I've made sure to include both types in the RN. But when a read-only mapping is being added, users don't need to be strictly told what type it appears as in Trino, because it will be evident the first time they query a ClickHouse table with a `DateTime(timezone)` column, and they can otherwise look in the docs or at the linked PR as necessary. There's no worries about anything breaking, nor do users need guidance beyond what's stated here to use this new functionality. I'm still open to changing it for consistency's sake, but this was a conscious decision made to prioritize conciseness where further info didn't seem required.

As far as communicating editorialization goes, that's probably worth a discussion. I'll follow-up with you on the Trino Slack - there's a few different ways we can go about this.

Reply:
As a general rule .. we editorialize all suggestions for the RN entry.If you want to confirm the ones for your merged PRs that would certainly be good, but we want to avoid pinging maintainers and wasting their valuable time with grammar edits and other things. Also keep in mind each RN is at minimum fully reviewed by @colebow @martint and myself

Reply:
> But when a read-only mapping is being added, users don't need to be strictly told what type it appears as in Trino, because it will be evident the first time they query a ClickHouse table with a DateTime(timezone) column, and they can otherwise look in the docs or at the linked PR as necessary.

While this is pragamatic it's not practically useful. Since you can CTAS/INSERT INTO SELECT which would get affected by the read mapping as well. It's useful information to have.

I'd not ask for it to be added if we linked to the type mapping docs from here but we don't. Let's not make things harder for users by having to manually search or know what the correct page is for the information they are seeking.